docs/api-reference/next.config.js/environment-variables.md

```diff
@@ -41,7 +41,7 @@ For example, the following line:
 return <h1>The value of customKey is: {process.env.customKey}</h1>
 ```
 
-Will end up being:
+will end up being:
```

Comment:
Should be uppercase probably as any paragraph start tho lower case is easier to read i guess